pkg/display/gitrepo.go

```diff
@@ -208,48 +196,16 @@ func appendState(states map[key][]fleet.ResourcePerClusterState, key key, state
 	}
 }
 
-func bundleResources(bundle *fleet.Bundle, isNSed func(schema.GroupVersionKind) bool) (map[key][]fleet.ResourcePerClusterState, []error) {
-	var (
-		errors          []error
-		bundleResources = map[key][]fleet.ResourcePerClusterState{}
-	)
-
-	m, err := manifest.New(&bundle.Spec)
-	if err != nil {
-		errors = append(errors, err)
-		return nil, errors
-	}
-
-	for _, target := range bundle.Spec.Targets {
-		opts := options.Calculate(&bundle.Spec, &target)
-		objs, err := helmdeployer.Template(bundle.Name, m, opts)
-		if err != nil {
-			errors = append(errors, err)
-			continue
-		}
-
-		for _, obj := range objs {
-			m, err := meta.Accessor(obj)
-			if err != nil {
-				errors = append(errors, err)
-				continue
-			}
-			key := key{
-				Namespace: m.GetNamespace(),
-				Name:      m.GetName(),
-			}
-			gvk := obj.GetObjectKind().GroupVersionKind()
-			if key.Namespace == "" && isNSed(gvk) {
-				if opts.DefaultNamespace == "" {
-					key.Namespace = "default"
-				} else {
-					key.Namespace = opts.DefaultNamespace
-				}
-			}
-			key.APIVersion, key.Kind = gvk.ToAPIVersionAndKind()
-			bundleResources[key] = []fleet.ResourcePerClusterState{}
+func bundleResources(bundle *fleet.Bundle) map[key][]fleet.ResourcePerClusterState {
+	bundleResources := map[key][]fleet.ResourcePerClusterState{}
+	for _, resourceKey := range bundle.Status.ResourceKey {
+		k := key{
```

Comment:
Can the key type just be ResourceKey now?  Can we delete the key type declaration?